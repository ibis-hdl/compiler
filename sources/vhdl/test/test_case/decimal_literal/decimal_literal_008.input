-- Real literals
3.14159_26

