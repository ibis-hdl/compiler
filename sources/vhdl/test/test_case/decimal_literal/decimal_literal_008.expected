(decimal_literal
  literal: 1.34E-12, type: real
)
